.github/workflows/scripts/linux/generate-cmake.sh

```diff
@@ -21,9 +21,8 @@ cmake                                       \
 -DCMAKE_BUILD_TYPE=Release                  \
 -DPACKAGE_MODE=TRUE                         \
 -DDISABLE_ADVANCE_SIMD=TRUE                 \
--DCMAKE_INSTALL_LIBDIR="/tmp/"              \
--DCMAKE_INSTALL_DATADIR="/tmp/"             \
--DCMAKE_INSTALL_DOCDIR="/tmp/PCSX2"         \
+-DDISABLE_PCSX2_WRAPPER=TRUE                \
+-DCMAKE_INSTALL_PREFIX="squashfs-root/usr/" \
```

Comment:
AppImage cannot see "squashfs-root" it translates to `/`
The lookup will resolve to an unknown path.

Reply:
This string is no longer embedded into the application, the application will get things like `../usr/share`

Reply:
Relative paths is much cleaner.
Then `Configuration_Guide.pdf` & `PCSX2_FAQ.pdf` should go in ```../usr/share/PCSX2/``` ?